src/services/base64Files/hooks/setDownloadHeaders.js

```diff
@@ -0,0 +1,9 @@
+module.exports = (req, res, next) => {
+	if (res.statusCode === 200 && req.query.download === 'true' && res.data.data) { // und get
+		res.setHeader('Content-Type', 'application/pdf');
+		res.setHeader('Content-Disposition', 'attachment; filename="Nutzungsbedingungen.pdf"');
```

Comment:
Why is it "Nutzungsbedingungen.pdf"?

Reply:
What should be the correct name?

Reply:
@CeEv Do we still need the Content-Disposition header?